x/evm/keeper/grpc_query_test.go

```diff
@@ -887,18 +887,6 @@ func (suite *KeeperTestSuite) TestTraceTx() {
 			},
 			expPass: false,
 		},
-		{
```

Comment:
I thought we were going to comment this out instead of deleting with an explanation of why we did this?